README.md

```diff
@@ -11,7 +11,7 @@ Misk is not ready for use. The API is not stable.
 Our [change log][changelog] has release history. API is subject to change. 
 
 ```kotlin
-implementation("com.squareup.misk:misk:0.13.0")
```

Comment:
I thought there were more places than here.

Reply:
Like this:
https://github.com/cashapp/misk/blob/be2c3513e0aedd43a7f7fa10f3dab7646c93acf3/CHANGELOG.md

Reply:
sure, done